src/lib.rs

```diff
@@ -581,27 +581,27 @@ fn parse_sdp_vector(lines: &[SdpLine]) -> Result<SdpSession, SdpParserError> {
     let mut sdp_session = SdpSession::new(version, origin, session);
     for (i, line) in lines.iter().enumerate().skip(3) {
         match *line {
-            SdpLine::Attribute { value: ref v } => sdp_session.add_attribute(v.clone()),
-            SdpLine::Bandwidth { value: ref v } => sdp_session.add_bandwidth(v.clone()),
-            SdpLine::Timing { value: ref v } => sdp_session.set_timing(v.clone()),
-            SdpLine::Media { .. } => sdp_session.extend_media(parse_media_vector(&lines[i..])?),
-            SdpLine::Origin { .. } |
-            SdpLine::Session { .. } |
-            SdpLine::Version { .. } => {
+            SdpLine::Attribute(ref v) => sdp_session.add_attribute(v.clone()),
+            SdpLine::Bandwidth(ref v) => sdp_session.add_bandwidth(v.clone()),
+            SdpLine::Timing(ref v) => sdp_session.set_timing(v.clone()),
+            SdpLine::Media(..) => sdp_session.extend_media(parse_media_vector(&lines[i..])?),
```

Comment:
It is possible to use `_` rather than `..`, which is what I usually see here. I don't know if there is one that is stylistically preferred.